### ISSUE TITLE
HexColorPicker  + HexAlphaColorPicker have`aria-required-attr` accessibility issue

### ISSUE THREAD

**Issue by ikerestes** [omgovich/react-colorful#213]
`eslint-plugin-jsx-a11y` that evaluates accessibility's issues in react applications is registering a violation on `HexColorPicker` and `HexAlphaColorPicker` components.

> 
>  div[aria-label="Color"]
> Critical
> Required ARIA attribute not present: aria-valuenow
> 
> Required ARIA attributes must be provided
> 

More info
https://dequeuniversity.com/rules/axe/4.7/aria-required-attr?application=axeAPI
